### PR TITLE
Distribution builds

### PR DESCRIPTION
**Pull request by alloy**
- [x] Proper server-side source-maps 
- [x] Proper client-side source-maps.
- [x] Remove babel plugin module resolver: https://github.com/tleunen/babel-plugin-module-resolver/issues/162
- [x] Test that everything works as expected after integration in Force.
- [x] Make dist build runnable. This is now also being used for a debugger task that I added to vscode.

### CODE REVIEW

**Comment by alloy** [comment]
Rebasing.

**Comment by alloy** [comment]
Done.

**Comment by l2succes** [comment]
Ok, tested the force integration. mergin'!

File: package.json [hunk 1]
```diff
@@ -43,7 +43,9 @@
     "babel-preset-es2015": "^6.22.0",
     "babel-preset-react": "^6.22.0",
     "babel-preset-stage-3": "^6.22.0",
-    "babel-relay-plugin": "https://github.com/alloy/relay/releases/download/v0.9.3/babel-relay-plugin-0.9.3.tgz",
+    "babel-relay-plugin": "0.9.3",
```
Comment (alloy) [inline package.json]: This isn’t right, it’s extremely important that we use our own fork so that our custom `__id` field works.

Comment (l2succes) [inline package.json]: Oh word, reverting

**Pull request merged**